File: app/src/main/java/in/ceeq/define/utils/AnalyticsUtils.java
package in.ceeq.define.utils;

import android.app.Application;


import in.ceeq.define.BuildConfig;

public class AnalyticsUtils {

    public void init(Application application) {
        if (!BuildConfig.DEBUG) {

        } else {
            // Uncomment if you want to use Stetho
            //Stetho.initializeWithDefaults(application);
        }
    }
}

File: app/src/main/java/in/ceeq/define/utils/AnalyticsUtils.kt
package `in`.ceeq.define.utils


import `in`.ceeq.define.BuildConfig
import android.app.Application

class AnalyticsUtils {

    fun init(application: Application) {
        if (!BuildConfig.DEBUG) {

        } else {
            // Uncomment if you want to use Stetho
            //Stetho.initializeWithDefaults(application);
        }
    }
}
